apps/src/p5lab/utils.js

```diff
@@ -38,7 +38,7 @@ export function parseSoundPathString(text) {
     category = pathStringArray[2];
     // Example: 'category_board_games' becomes 'Board games: '
     category = category.replace('category_', '');
-    category = category.replaceAll('_', ' ');
+    category = category.replace(/_/g, ' ');
```

Comment:
Clever idea to use `replace` with a regular expression instead. It seems like this would still replace all underscore characters in the string, is that right? For example, if we had `'category_fruits_and_vegetables'`, would the result be as expected? No category name in the current library is made of more than two English words, but I could imagine that changing at some point.

Reply:
Yes, this replaces all underscores with white spaces. For `category_fruits_and_vegetables` the result would be `Fruits and vegetables`. This is what I would think is expected. You?